eng/pipelines/eng-common-sync.yml

```diff
@@ -13,27 +20,83 @@ pr:
 pool:
   vmImage: windows-2019
 
-jobs:
-  - job: SyncEngCommon
-    displayName: Sync eng/common Directory
-
-    steps:
-    - template: ./templates/steps/sync-directory.yml
-      parameters:
-        CommitMessage: "Sync eng/common directory with azure-sdk-tools repository for Tools PR https://github.com/Azure/azure-sdk-tools/pull/$(System.PullRequest.PullRequestNumber)"
-        DirectoryToSync: "eng/common"
-        PRBranchName: "sync-eng-common-$(System.PullRequest.SourceBranch)-$(System.PullRequest.PullRequestNumber)"
-        PRTitle: "Sync eng/common directory with azure-sdk-tools for PR $(System.PullRequest.PullRequestNumber)"
-        PRBody: >- 
-          Sync eng/common directory with azure-sdk-tools for PR https://github.com/Azure/azure-sdk-tools/pull/$(System.PullRequest.PullRequestNumber) <br>
-          See [eng/common workflow](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/README.md#workflow)
-        Repos:
-          - azure-sdk-for-android
-          - azure-sdk-for-c
-          - azure-sdk-for-cpp
-          - azure-sdk-for-go
-          - azure-sdk-for-ios
-          - azure-sdk-for-java
-          - azure-sdk-for-js
-          - azure-sdk-for-net
-          - azure-sdk-for-python
+stages:
+  - stage: CreateSync
+    jobs:
+      - job: SyncEngCommon
+        displayName: Sync eng/common Directory
+
+        steps:
+        - pwsh: |
+            $DirectoryToSync = "eng/common"
+            if ('$(Build.Reason)' -eq 'PullRequest') 
+            {
+              $CommitMessage = "Sync eng/common directory with azure-sdk-tools repository for Tools PR $(System.PullRequest.PullRequestNumber)"
```

Comment:
```suggestion
              $CommitMessage = "Sync $(DirectoryToSync) directory with azure-sdk-tools repository for Tools PR $(System.PullRequest.PullRequestNumber)"
```

Reply:
Hopefully we can end up making the commit message match the commit message of the Tools PR but that can happen in a later change when we update the syncing logic.